openruth.module

```diff
@@ -433,13 +433,20 @@ function _openruth_user_status($creds = NULL, $reset = FALSE) {
  */
 function openruth_seek_reserved_item($faust_number) {
   global $user;
+
+  if (!function_exists('openruth_reservation_list')) {
+    include_once(drupal_get_path('module', 'openruth') . '/includes/openruth.reservation.inc');
+  }
```

Comment:
#bug551
http://platform.dandigbib.org/issues/551

For some reason reservation.inc was not included when handling ILL loans.
Probably because ting client throws an exception like this one:
'Unexpected error message in response'
that one comes from opensearch:
'"Error: unknown/missing record: 870970-basis:575023933V"
and that one comes from the V appended to the pid. That can happen in the
local system.

On Tue, Jan 13, 2015 at 1:46 PM, Jesper Kristensen <notifications@github.com
> wrote:

> In openruth.module
> <https://github.com/ding2/openruth/pull/5#discussion-diff-22859263>:
>
> > @@ -433,13 +433,20 @@ function _openruth_user_status($creds = NULL, $reset = FALSE) {
> >   */
> >  function openruth_seek_reserved_item($faust_number) {
> >    global $user;
> > +
> > +  if (!function_exists('openruth_reservation_list')) {
> > +    include_once(drupal_get_path('module', 'openruth') . '/includes/openruth.reservation.inc');
> > +  }
>
> Changed it to use module_load_include. Why it's need we need to ask
> @pjohans <https://github.com/pjohans>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ding2/openruth/pull/5/files#r22859263>.
>